pkg/cmd/kind/completion/completion.go

```diff
@@ -69,6 +71,9 @@ This depends on the bash-completion binary.  Example installation instructions:
 # for fish users
 	% kind completion fish > ~/.config/fish/completions/kind.fish
 
+# for powershell users
+	PS> kind completion powershell | Out-String | Invoke-Expression
```

Comment:
for reference: https://github.com/spf13/cobra/blob/main/site/content/completions/_index.md#creating-your-own-completion-command